test/built-ins/Map/prototype/getOrInsertComputed/returns-value-if-key-is-present-different-key-types.js

```diff
@@ -16,40 +16,35 @@ features: [Symbol, arrow-function, upsert]
 ---*/
 var map = new Map();
 
-map.set('bar', 0);
-assert.sameValue(map.get('bar'), map.getOrInsertComputed('bar', () => 1));
-assert.sameValue(0, map.getOrInsertComputed('bar', () =>  1));
+var item = 'bar';
+map.set(item, 0);
+assert.sameValue(0, map.getOrInsertComputed(item, () =>  1));
```

Comment:
Please swap the arguments to `assert.sameValue` (throughout the file), and one space after `=>` is enough